core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -228,7 +243,10 @@ internal class CacheFallbackOperations
             }
             catch (Exception ex)
             {
-                CoreLoggerBase.Default.Warning("GetAllAdalEntriesForMsal failed due to Exception - " + ex);
+                string msg = "GetAllAdalEntriesForMsal failed due to Exception - ";
```

Comment:
>GetAllAdalEntriesForMsal [](start = 30, length = 24)

should probably be more readable as well. E.g.: s: An error occurred while searching for valid MSAL cache entries in ADAL cache.